BraveRewards.xcodeproj/project.pbxproj

```diff
@@ -1556,6 +1580,7 @@
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 4.2;
 				TARGETED_DEVICE_FAMILY = "1,2";
+				TEST_HOST = "$(BUILT_PRODUCTS_DIR)/BraveRewardsExample.app/BraveRewardsExample";
```

Comment:
I needed it originally when I was running the tests because it was looking for the database store at a URL that doesn't exist without an app bundle (Documents for instance)...

Perhaps since I ported the `InMemoryDataController` it no longer needs it